datafusion/physical-plan/src/sorts/partial_sort.rs

```diff
@@ -647,157 +591,6 @@ mod tests {
         Ok(())
     }
 
-    fn prepare_partitioned_input() -> Arc<dyn ExecutionPlan> {
```

Comment:
I deleted these test case for the same reason, the sort in ExternalSort will merge the RecordBatchs so I think these test cases are not necessary?